index.html

```diff
@@ -21,11 +21,158 @@ <h2 class="t2">I’m a software developer</h2></div>
+                <button class="js" type="button">javScript</button>
+                <button class="html" type="button">html</button>
+            </li>
+            <li><button class="projects" type="button">See Projects</button></li>
+        </ul>
+    </div>
+
+    <div class="card">
+        <ul class="box">
+            <li><h4>Multi-Post Stories Gain+Glory</h4></li>
+            <li class="buttons">
+                <button class="ror" type="button">Ruby on rails</button>
+                <button class="css" type="button">css</button>
+                <button class="js" type="button">javScript</button>
+                <button class="html" type="button">html</button>
+            </li>
+            <li><button class="projects" type="button">See Projects</button></li>
+        </ul>
+    </div>
+</section>
+    <section class="about">
+    <div class="aboutme">
+        <h2>About me</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <button class="resume" type="button">Get My Resume</button>
+    </div>
+
+    <div class="lang">
+        <img src="./images/icons/icon-languages.png" alt="icons">
+        <h3>Languages</h3>
+        <ul class="L">
+            <li class="js0"><a>JavaScript</a></li>
+            <li class="rub"><a>Ruby</a></li>
+            <li class="ht"><a>Html</a></li>
+            <li class="cs"><a>CSS</a></li>
+        </ul>
+    </div>
+
+    <div class="frameWorks">
+        <img src="./images/icons/icon-frameworks.png" alt="icons">
+        <h3>Frameworks</h3>
+        <ul class="fw1">
+            <li class="bs"><a>Bootstrap</a></li>
+            <li class="rubr"><a>Ruby on rails</a></li>
+            <li class="rs"><a>RSpec</a></li>
+        </ul>
+        <ul class="fw2">
+            <li class="capy"><a>Capybara</a></li>
+            <li class="sele"><a>selenium</a></li>
+        </ul>
+    </div>
+
+    <div class="skills">
+        <img src="./images/icons/icon-skills.png" alt="icons">
+        <h3>Skills</h3>
+        <ul class="sk">
+            <li class="codek"><a>Codekit</a></li>
+            <li class="gith"><a>GitHub</a></li>
+            <li class="codep"><a>Codepen</a></li>
+            <li class="gitl"><a>Gitlab</a></li>
+        </ul>
+            <div class="ter"><a>Terminal</a></div>
+    </div>
+    </section>
+<section class="contact">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contactForm" action="https://formspree.io/f/mgedpeao" method="post">
+        <input class="fName" name="fname" type="text" placeholder="Full name" maxlength="30" required>
+        <input class="email" name="email" type="email" placeholder="Email address" required>
+        <textarea class="message" name="message" required>Enter text here</textarea>
```

Comment:
- [ ] Hey, your `text area` is missing the `required attribute` and `max-length attribute`. kindly add them.